### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chinmaypurav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <p align="center"><a href="https://laravel.com" target="_blank"><img src="https://raw.githubusercontent.com/laravel/art/master/logo-lockup/5%20SVG/2%20CMYK/1%20Full%20Color/laravel-logolockup-cmyk-red.svg" width="400" alt="Laravel Logo"></a></p>
 
+
+Test
```
Comment (amitavroy) [inline README.md:4]: Why test?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <p align="center"><a href="https://laravel.com" target="_blank"><img src="https://raw.githubusercontent.com/laravel/art/master/logo-lockup/5%20SVG/2%20CMYK/1%20Full%20Color/laravel-logolockup-cmyk-red.svg" width="400" alt="Laravel Logo"></a></p>
```
Comment (amitavroy) [inline README.md:1]: Can we make it left?

Comment (amitavroy) [inline README.md:1]: Change logo

**Review by amitavroy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chinmaypurav) [reply README.md:4]: or if we can change it it test new?

File: README.md (hunk 2 above)
Comment (chinmaypurav) [reply README.md:1]: Not required

**Pull request merged**